app/webpack/stylesheets/_shame.scss

```diff
@@ -869,3 +869,8 @@ table#download-files-report {
 .error {
   color: $error-colour;
 }
+
+#unused-materials-over-3-hours .js-hidden {
```

Comment:
i reckon its better to just not use that `js-hidden` class when it is not needed - this could just lead to confusion

Reply:
so i would not have it in the shared partial. if you need class for styling purposes i would create a separate one for clarity, no?!